src/appleseed/renderer/modeling/aov/aov.cpp

```diff
@@ -163,6 +163,8 @@ void UnfilteredAOV::create_image(
             tile_height,
             num_channels,
             PixelFormatFloat);
+
+    clear_image();
```

Comment:
Whyu does `UnfilteredAOV::create_image()` clears the image but `AOV::create_image()` doesn't? Can we make the behavior more consistent?

Reply:
There are 2 reasons. One is that for some AOVs, 0 is not the default channel value (normals for example).
The other is that there is an extra hidden channel in in the images of unfiltered AOVs.
The accumulators use it to keep track of the nearest sample to the pixel center seen so far.

We can clear all the images, but it is not needed for the others AOVs and by clearing them we will allocate all the tiles in the image.

Reply:
Thanks for the explanation. Can you add it as a comment in the code?

Reply:
Fixed and rebased